newsroom/company_admin/views.py

```diff
@@ -51,28 +57,28 @@ def get_view_data():
     }
 
 
-@blueprint.route("/company_admin/send_product_seat_request", methods=["POST"])
-@login_required
-@company_admin_only
-async def send_product_seat_request_email():
+@blueprint.endpoint("/company_admin/send_product_seat_request", methods=["POST"], auth=[auth_rules.company_admin_only])
+async def send_product_seat_request_email(request: Request):
     user = get_user_from_request(None)
     company = get_company_from_request(None)
 
-    if not company:
-        return NotFound(gettext("Company not found"))
+    assert company is not None
 
-    data = await get_json_or_400()
+    if not company:
+        await request.abort(404, gettext("Company not found"))
```

Comment:
Would this code block never be reached due to the above `assert` statement?

Reply:
yeah, indeed. I moved it down as it is needed to avoid mypy complains.